src/BlazorAdmin/Shared/RedirectToLogin.razor

```diff
@@ -1,9 +1,9 @@
 ﻿@inject NavigationManager Navigation
-@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
+
 @code {
     protected override void OnInitialized()
     {
-        Navigation.NavigateTo($"authentication/login?returnUrl=" +
+        Navigation.NavigateTo($"Identity/Account/Login?returnUrl=" +
```

Comment:
Are these URIs/paths case insensitive? Ideally they should be.

Reply:
yes it is case insensitive

Reply:
OK